.github/workflows/ci_cd.yml

```
@@ -218,7 +215,7 @@ jobs:
 
   release:
     if: github.event_name == 'refs/heads/main' && !contains(github.ref, 'refs/tags')
-    needs: [style, doc-build] #docs-style
+    needs: [code-style, doc-build] #docs-style
```

```suggestion
    needs: [doc-build]
```
``doc-build`` already depends on ``code-style`` and ``docs-style``